docs/distributing-policies/publish-policy-to-artifact-hub.md

```diff
@@ -0,0 +1,110 @@
+host your policies.
+:::
+
+## Prepare your Git repository
+
+Artifact Hub crawls Git repositories looking for special metadata files.
+
+There are different kind of layouts the Git repository can have. They
+are all documented in depth by the [official Artifact Hub documentation](https://artifacthub.io/docs/topics/repositories/#kubewarden-policies-repositories).
+
+Artifact Hub is pretty flexible, you can organize your code in these ways:
+
+* Have a Git repository dedicated to Artifact Hub: this repository will not contain
+  any policy source code. It will be a collection of the YAML files required by
+  Artifact Hub.
+* Add a Artifact Hub directory inside of the Git repository that holds the source
+  of your policy. This is an iteration of the previous approach, the only difference
+  is that it focuses just on one policy: the one defined inside of the Git repository.
+  This approach allows to keep multiple versions of the policy published on Artifact Hub.
+* Add the `artifacthub-pkg.yml` and the `artifacthub-repo.yml` files to the root
+  of the Git repository that holds the policy source code. This approach is the 
+  simplest one. The only limitation is that only the latest version of the policy
+  will be visible on Artifact Hub.
+
+The last approach is what is used inside of our official policy templates.
+The Git repository that is scaffolded will include the `artifacthub-pkg.yml`
+and `artifacthub-repo.yml` files.
+
+## Publishing Steps
+
+Before publishing a policy to Artifact Hub, you must create an account on
+the [website](https://artifacthub.io/). To do that, use the *"Sign Up"* button
+at the top of the page.
+
+Before publishing the policy, ensure your Git repository has the proper layout.
+
+If you created the policy using one of our templates, ensure the `artifacthub-pkg.yml` file
+has correct data inside of it.
+The format of the `artifacthub-pkg.yml` is described [here](https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml).
+
+Finally, ensure your policy is published inside of a container registry or on a
+web server.
+
+:::note
+Right now the contents of the `artifacthub-repo.yml` file are not relevant.
+:::
+
+Once everything is in place, log into Artifact Hub and go to your
+[control plane](https://artifacthub.io/control-panel/repositories?page=1).
+
+Decide whether you want to publish the policy as an user or under an Artifact Hub
+organization you belong to. This is done by choosing the right *"control panel context"*.
+
+Then press the *"Add"* button and fill the form:
+
+  * Choose *"Kubewarden policies"* as kind.
+  * Enter a *"Name"* and *"Display name"* of your choice.
+  * Enter the URL to your Git repository.
+
+Finally, press the *"Add"* button. This will bring you back to the *"Repositories"*
+page, where you will see your freshly created repository.
+
+Each repository has a series of information. Find the *"ID"* property of the
```

Comment:
Nit:

```suggestion
Each repository has several information fields. Find the *"ID"* property of the
```

Or something similar, it's just that here the plural + singular of `series of information` sounds weird.